src/BeadingStrategy/SingleBeadBeadingStrategy.cpp

```diff
@@ -40,15 +40,30 @@ coord_t SingleBeadBeadingStrategy::getOptimalThickness(coord_t bead_count) const
 
 coord_t SingleBeadBeadingStrategy::getTransitionThickness(coord_t lower_bead_count) const
 {
-    if (lower_bead_count <= 0) return 0;
-    else if (lower_bead_count == 1) return optimal_width;
-    else return 9999999;
+    if (lower_bead_count <= 0) 
+    {
+        return 0;
+    }
+    else if (lower_bead_count == 1) 
+    {
+        return optimal_width;
+    }
+    else
+    {
+        return 9999999;
```

Comment:
Change to `std::numeric_limits<coord_t>::max()` (you'll need to import `<limits>`).

Reply:
@nallath 's latest commit fixes this, closing.